include/deal.II/grid/tria_accessor.h

```diff
@@ -3135,6 +3139,32 @@ class CellAccessor : public TriaAccessor<dim, dim, spacedim>
   CellAccessor<dim, spacedim> &
   operator=(CellAccessor<dim, spacedim> &&) = default; // NOLINT
 
+  /**
+   * @}
+   */
+
+  /**
+   * @name Converting iterators
+   */
+  /**
+   * @{
+   */
+
+  /**
+   * A function that converts a Triangulation active cell iterator to a
+   * DoFHandler active cell iterator, or a DoFHandler active cell iterator
+   * to an active cell iterator of another DoFHandler.The @p iterator must be
+   * associated with the triangulation of the @p dof_handler.
+   *
+   * @param dof_handler The DoFHandler for the output active cell iterator.
+   * @return An active cell iterator for the @p dof_handler, matching the cell
+   *         referenced by the input @p iterator. The concrete type of the
+   *         returned object is a DoFHandler::active_cell_iterator.
+   */
+  auto
+  convert_to_dof_handler_iterator(
```

Comment:
You could also call it
```suggestion
  as_dof_handler_iterator(
```
(but the current choice is fine as well)

Reply:
Changed! Thanks for the recommendation.